src/topics/recent.js

```diff
@@ -63,6 +64,10 @@ module.exports = function(Topics) {
 
 	Topics.updateRecent = function(tid, timestamp, callback) {
 		callback = callback || function() {};
-		db.sortedSetAdd('topics:recent', timestamp, tid, callback);
+		if (plugins.hasListeners('filter:topics.updateRecent')) {
+			plugins.fireHook('filter:topics.updateRecent', {tid: tid, timestamp: timestamp}, callback);
```

Comment:
Can you change this so it doesn't return the data.

```
if (plugins.hasListeners('filter:topics.updateRecent')) {
    plugins.fireHook('filter:topics.updateRecent', {tid: tid, timestamp: timestamp}, function(err, data) {
        callback(err);
    });
}
```

`Topics.updateRecent` should only return back `err`